views/pages/StudiesPage/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react'
+
+import { Box } from '@mui/material'
+
+import { SORT_DIRECTION } from '../../../constants'
+import api from '../../api'
+import PageHeader from '../../components/PageHeader'
+import useTableSort from '../../hooks/useTableSort'
+import StudiesTable from '../../tables/StudiesTable'
+
+const site = 'site'
+const study = 'study'
+
+const StudiesPage = () => {
+  const [studies, setStudies] = useState([])
+  const { onSort, sortDirection, sortBy } = useTableSort('study')
+  const loadStudies = async () => {
+    const sortParams = {
+      ...(sortBy ? { sortBy: sortBy === site ? study : sortBy } : {}),
```

Comment:
why are we interchanging study and site? Can we use the same terminology?

Reply:
This is the approach that I went with for the following reasons.

The sitemetada csv's do not have a siteName property, we have a list of them in the code, but with the latest dataset there appears to be one or two names missing, to reduce any errors of missing data, the sort by siteName is done by using the study value, aka the site's short code. 

Ideally we should store the store the site name, but one we wont always have an updated list of site names, which would need to be maintained, or the client has to include that in their csv's.

Reply:
[112650b](https://github.com/AMP-SCZ/dpdash/pull/723/commits/112650bca788220e00e45c0c6cca236a02c75414)
updated the name of the variables